_observing-your-data/alerting/monitors.md

```diff
@@ -138,7 +138,12 @@ POST _nodes/reload_secure_settings
 1. Specify a name for the monitor.
 1. Choose either **Per query monitor**, **Per bucket monitor**, **Per cluster metrics monitor**, or **Per document monitor**.
 
-Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each of those buckets can trigger an alert, but query-level monitors can only trigger one alert at a time.
+OpenSearch supports the following types of monitors:
+
+- **Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time. 
```

Comment:
Should "Query-level monitors" be "Per query monitors"?

Reply:
This should be "Per query monitors".